specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -866,6 +882,23 @@
         "requireInfrastructureEncryption": {
           "$ref": "#/definitions/WorkspaceCustomBooleanParameter",
           "description": "A boolean indicating whether or not the DBFS root file system will be enabled with secondary layer of encryption with platform managed keys for data at rest."
+        },
+        "storageAccountName": {
+          "$ref": "#/definitions/WorkspaceCustomStringParameter",
+          "description": "Default DBFS storage account name."
+        },
+        "storageAccountSkuName": {
+          "$ref": "#/definitions/WorkspaceCustomStringParameter",
+          "description": "Storage account SKU name, ex: Standard_GRS, Standard_LRS. Refer https://aka.ms/storageskus for valid inputs."
+        },
+        "vnetAddressPrefix": {
+          "$ref": "#/definitions/WorkspaceCustomStringParameter",
+          "description": "Address prefix for Managed virtual network. Default value for this input is 10.139."
+        },
+        "resourceTags": {
+          "$ref": "#/definitions/WorkspaceCustomObjectParameter",
+          "readOnly": true,
+          "description": "List of tags applied to managed resources under MRG. These can be updated by updating tags at workspace level."
```

Comment:
Since this is readonly field,  "**Apply Tags to resources under Managed resource group. These can be updated by updating tags at workspace level".**

I kept same message as-is just expand MRG